mysqli/mysqli.php

```diff
@@ -141,7 +142,7 @@ public function __construct (
 		$username = null,
 		$passwd = null,
 		$dbname = null,
-		$port = null,
+		$port = 0,
```

Comment:
according to https://github.com/php/php-src/blob/master/ext/mysqli/mysqli.stub.php 
```
    public function __construct(
        ?string $host = null,
        ?string $user = null,
        ?string $password = null,
        ?string $database = null,
        ?int $port = null,
        ?string $socket = null
    ) {}
```
Could you please clarify why you have changed this value?